database_admin/migrations/058_mark_stale_fix.up.sql

```diff
@@ -0,0 +1,92 @@
+CREATE OR REPLACE FUNCTION delete_systems(inventory_ids UUID[])
+    RETURNS INTEGER
+AS
+$$
+DECLARE
+    tmp_cnt INTEGER;
+BEGIN
+
+    WITH systems as (
+        SELECT rh_account_id, id
+        FROM system_platform
+        WHERE inventory_id = ANY (inventory_ids)
+        ORDER BY rh_account_id, id FOR UPDATE OF system_platform),
+         marked as (
+             UPDATE system_platform sp
+                 SET opt_out = true
+                 WHERE (rh_account_id, id) in (select rh_account_id, id from systems)
+         ),
+         advisories as (
+             DELETE
+                 FROM system_advisories
```

Comment:
It would look better to be aligned :wink:

Reply:
Intellij formatter changes it to this.